.drone.yml

```diff
@@ -586,7 +586,7 @@ steps:
   - cd $WORKSPACE_DIR/go/src/github.com/gravitational/teleport
   - build.assets/build-fido2-macos.sh build
   - export PKG_CONFIG_PATH="$(build.assets/build-fido2-macos.sh pkg_config_path)"
-  - make clean release OS=$OS ARCH=$ARCH FIDO2=yes TOUCHID=yes PIV=yes
+  - make clean release-darwin OS=$OS ARCH=$ARCH FIDO2=yes TOUCHID=yes PIV=yes
```

Comment:
Is there any point in setting `OS` anymore if the target is called `release-darwin`? Seems error prone to have 2 ways to set the OS.

Reply:
This change is leftover from some previous testing. Reverted this and most of the dronegen changes.